util/storage-samples/storage-rv-sample/src/test/java/org/fido/iot/storage/RvsStorageTest.java

```diff
@@ -32,7 +39,9 @@
 import org.fido.iot.protocol.To1ClientStorage;
 import org.fido.iot.protocol.To1ServerService;
 import org.fido.iot.protocol.To1ServerStorage;
+import org.junit.jupiter.api.TestMethodOrder;
 
+@TestMethodOrder(Alphanumeric.class)
```

Comment:
Could you please help me understand why we need this annotation?

Reply:
To execute the tests in Alphanumeric order.